test/integration/case.test.js

```diff
@@ -183,6 +183,7 @@ describe('Case', () => {
         results.body.should.have.property('case');
         results.body.case.should.be.a('object');
         results.body.case.should.have.property('caseId');
+        results.body.case.should.have.property('contact_tracer_id');
```

Comment:
We should be returning this value in its camelcase format. Mind updating the `_mapCase` function here to do that? https://github.com/Path-Check/safeplaces-backend/blob/dev/db/models/cases.js#L220

Reply:
Fixed